### ISSUE TITLE
Consider using non-proprietary fonts

### ISSUE THREAD

**Issue by tingerrr** [jneug/typst-mantys#18]
Consider using a free equivalent for the proprietary font fallbacks use in the default config.

Especially in CI, where one would usually generate documentation automatically, installing fonts is a pain when they're not free. In general, the default theme should not assume a proprietary font is or can be installed, even if it is just a fallback.

A very dead-on free equivalent to `Helvetica` is `Tex Gyre Heros`.
![image](https://github.com/jneug/typst-mantys/assets/137803093/e804f108-fccb-472a-bdbd-8bb4ede5fdf5)

**Comment by jneug** [comment]
Seems easy enough.

**Comment by tingerrr** [comment]
Fixed in 53c59806 it seems? I'm still getting the warning on 1.0.0 and I'm unsure why that is. I get the warning even when I change the font to just TeX Gyre Heros with `create-theme`:

```typst
#show: mantys.mantys(
  theme: mantys.create-theme(
    fonts: (sans: "TeX Gyre Heros"),
    heading: (font: "TeX Gyre Heros"),
  ),
  ...
)
```

Not sure why that is, I think this can be closed though.

**Comment by jneug** [comment]
🤔 I will habe a look at this. Not sure where the warning comes from.

**Comment by tingerrr** [comment]
The warning comes from [src/themes/default.typ:236](https://github.com/jneug/typst-mantys/blob/96e49580885c201d3ee44d70d8d9982f00c8421e/src/themes/default.typ#L236), because it uses `base` instead of `theme`.

I had a fix for this locally, but the tests needed updating and had some missing ignore files (possibly my fault tbh, the ignore file handling wasn't very good on `typst-test` until recently).

I suggest updating to `tytanic 0.1.0` for tests, then you can run `tt util vcs ignore` to regenerate the correct ignore files and then update whichever tests should be updated, some of them contain the date in their references which is not a good idea, this will always cause them to fail after one day.

**Comment by jneug** [comment]
Good catch, will update the default theme. Need to  publish an update with the new Tidy version anyways.

I need to have a thorough look at tytanic. Congrats on the release, by the way. 😃
